### PR TITLE
HLM-638 | verify revoke and suspended certificates

### PR DESCRIPTION
**Pull request by ajay-beehyv**
@prasanna-egov @saiprakash-v @varadeth

### CODE REVIEW

File: verification_app/src/config.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ const CERTIFICATE_SCAN_TIMEOUT = process.env.REACT_APP_CERTIFICATE_SCAN_TIMEOUT
 const CERTIFICATE_SIGNED_KEY_TYPE = process.env.REACT_APP_CERTIFICATE_SIGNED_KEY_TYPE || 'RSA';
 const certificatePublicKeyBase58 = process.env.REACT_APP_CERTIFICATE_PUBLIC_KEY_BASE58 || "DaipNW4xaH2bh1XGNNdqjnSYyru3hLnUgTBSfSvmZ2hi";
 
+const CERTIFICATE_STATUS_VC = process.env.REACT_CERTIFICATE_STATUS_VC || 'true';
```
Comment (varadeth) [inline verification_app/src/config.js:14]: have you tested this?
also it should be REACT_APP_CERTIFICATE_STATUS_VC

File: verification_app/src/components/VcCertificateStatus/index.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            }
+
+        }
+
+        verifyData()
+
+    }, []);
+
+    async function verifyCertificate(data) {
+        return axios.post("/vc-certification/v1/certificate/verify",data,{headers: {Authorization: ""}})
```
Comment (varadeth) [inline verification_app/src/components/VcCertificateStatus/index.js:102]: headers should not be needed here

**Review by varadeth — CHANGES_REQUESTED** [review]

File: verification_app/src/config.js (hunk 1 above)
Comment (ajay-beehyv) [reply verification_app/src/config.js]: Haven't expected it, resolved

**Review by varadeth — APPROVED** [review]

**Comment by prasanna-egov** [comment]
Please resolve conflicts and update the libraries.

**Comment by ajay-beehyv** [comment]
> Please resolve conflicts and update the libraries.

**Pull request merged**